.gitignore

```diff
@@ -5,3 +5,5 @@ lib
 /npm-packages-offline-cache
 tmp
 yarn
+/package-lock.json
+/.vscode
```

Comment:
`package-lock.json` is an npm thing, if you use npm you're not using the yarn lockfile so transitive dependencies could drift outside of what we're using. There should not be a package-lock.json file in your environment.